models/dex/dex_schema.yml

```diff
@@ -63,6 +63,7 @@ models:
         tests:
           - dbt_utils.accepted_range:
               max_value: 1000000000 # $1b is an arbitrary number, intended to flag outlier amounts early
+              severity: warn
```

Comment:
I add a severity setting here.

Reply:
nice. i think we should revert, since we run this test in prod and want errors to appear and send alerts to us